jetstream/core/server_lib.py

```diff
@@ -145,6 +137,21 @@ def run(
         "Not starting Prometheus server: --prometheus_port flag not set"
     )
 
+  logging.info("Kicking off gRPC server.")
```

Comment:
Looks like you only need the "get_model_load_time_metric()" change, any reason to move all gRPC server code?

Reply:
Actually just moved the prometheus server startup code to before the grpc server code since the metric tracks the time it takes for part of it to run

Reply:
sounds good, thanks.